src/header/js/header-menus.js

```diff
@@ -163,14 +163,23 @@ export default ({variant = 'desktop'}) => {
 
 								const hasStructured = hasCols && item.cols.filter((col) => (col.type === 'structured')).length > 0;
 								const structuredCols = hasCols ? item.cols.length : 0;
+								let multiCols = 0;
+
+								if (item.menus && item.menus.length > 10) {
+									(parseInt(item.menus.length % 3) === 0) ? multiCols = true : multiCols = false;
```

Comment:
do you have to use `parseInt` here? If menus is an array length will be an int. Also, you're mixing boolean and integer for `multiCols` var. Better way would be to do;

```
let multiCols = false;
if (item.menus && item.menus.length > 10) {
 multiCols = item.menus.length % 3 === 0;
}
```

Reply:
@paulcpederson this has been updated.